### PR TITLE
[MODULAR] Adds the pirate eyepatch to crafting menu

### PR DESCRIPTION
**Pull request by Mega-Drones**
## About The Pull Request
adds the pirate eyepatch as a craftable item for one cloth in the clothing category of the crafting menu

## How This Contributes To The Skyrat Roleplay Experience
The black pirate eyepatch currently can only be acquired in-game by either spawning with one in your loadout or finding it on the corpse of a space-pirate; effectively barring ghost roles from being able to get it.
This pull request fixes this issue, as well as adds a way to finally craft eyepatch huds

## Changelog

:cl: MegaDrone

qol: Adds a way for ghost roles to get the pirate eyepatch without needing to kill space-pirates, and allows for easier crafting of eyepatch huds

/:cl:

### CODE REVIEW

**Review by tf-4 — CHANGES_REQUESTED** [review]
Put it with the other one in code/game/objects/items/stacks/sheets/sheet_types.dm for consistency

**Comment by Mega-Drones** [comment]
Not sure why port tarkon is failing, considering I didn't touch anything that isn't part of cloth and crafting

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -379,6 +379,7 @@ GLOBAL_LIST_INIT(bamboo_recipes, list ( \
 GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("white jumpskirt", /obj/item/clothing/under/color/jumpskirt/white, 3), /*Ladies first*/ \
 	new/datum/stack_recipe("white jumpsuit", /obj/item/clothing/under/color/white, 3), \
+	new/datum/stack_recipe("Eyepatch", /obj/item/clothing/glasses/eyepatch, 1), \
```
Comment (tf-4) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
	new/datum/stack_recipe("Eyepatch", /obj/item/clothing/glasses/eyepatch, 1), /* SKYRAT EDIT ADDITION*/ \
```
Move it to below the other eyepatch on line 410 also. Then we're good.

**Review by tf-4 — CHANGES_REQUESTED** [review]
Don't worry about the port tarkon thing

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (tf-4) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: (and the comment)

**Review by tf-4 — APPROVED** [review]

**Pull request merged**